Documentation/conf.py

```diff
@@ -175,7 +175,7 @@
 spelling_filters = ["cilium_spellfilters.WireGuardFilter"]
 
 # Ignore some warnings from MyST parser
-suppress_warnings = ['myst.header', 'myst.xref_missing']
+suppress_warnings = ["myst.header", "myst.xref_missing"]
```

Comment:
This seems unrelated? Probably because this line was edited recently and your initial version would add the same warning exception? Let's keep the quotes, the file already uses a mix of both anyway.